SCClassLibrary/Common/Control/SkipJack.sc

```diff
@@ -45,9 +45,9 @@ SkipJack {
 	play { this.start }
 
 	stop {
-		task.stop;
 		all.remove(this);
 		CmdPeriod.remove(this);
 		if( verbose ) { ("SkipJack" + name + "stopped.").postcln };
+		task.stop;
```

Comment:
if we want to be exact, it's probably better to swap lines 50 and 51. If `task.stop` fails, the post will not lie.

Reply:
ahm, no ;-) the problem is: 
the task stops itself from inside when stopTest becomes true; 
calling task.stop inside a task stops it right there, so the post would only happen if stopped from outside, not from inside. 
we could change it to ("SkipJack" + name + "stopping.") if you prefer.

Reply:
yes, please! Then your explanation becomes much less necessary ;)

Reply:
ok done!